src/Tests/dotnet-sln.Tests/GivenDotnetSlnAdd.cs

```diff
@@ -1186,6 +1186,48 @@ public void WhenSolutionFolderAndInRootIsPassedItFails()
                 .Should()
                 .BeVisuallyEquivalentTo(contentBefore);
         }
+        
+        [Fact]
+        public void WhenSolutionFolderIsPassedWithDirectorySeparatorFolderStructureIsCorrect()
```

Comment:
can you collapse these tests into one by parameterizing using `[InlineData..]` ?